crates/scheduler/src/consumer/types/process_data.rs

```diff
@@ -53,3 +51,43 @@ impl Default for PaymentMethodsPTMapping {
         }
     }
 }
+
+/// Configuration for outgoing webhook retries.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct OutgoingWebhookRetryProcessTrackerMapping {
+    /// Default (fallback) retry configuration used when no merchant-specific retry configuration
+    /// exists.
+    pub default_mapping: RetryMapping,
+
+    /// Merchant-specific retry configuration.
+    pub custom_merchant_mapping: HashMap<String, RetryMapping>,
+}
+
+impl Default for OutgoingWebhookRetryProcessTrackerMapping {
+    fn default() -> Self {
+        Self {
+            default_mapping: RetryMapping {
+                // 1st attempt happens after 1 minute
+                start_after: 60,
+
+                frequency: vec![
+                    // 2nd and 3rd attempts happen at intervals of 5 minutes each
+                    60 * 5,
+                    // 4th, 5th, 6th, 7th and 8th attempts happen at intervals of 10 minutes each
+                    60 * 10,
+                    // 9th, 10th, 11th, 12th and 13th attempts happen at intervals of 1 hour each
+                    60 * 60,
+                    // 14th, 15th and 16th attempts happen at intervals of 6 hours each
+                    60 * 60 * 6,
+                ],
+                count: vec![
+                    2, // 2nd and 3rd attempts
+                    5, // 4th, 5th, 6th, 7th and 8th attempts
+                    5, // 9th, 10th, 11th, 12th and 13th attempts
+                    3, // 14th, 15th and 16th attempts
+                ],
```

Comment:
Since this concerns the `RetryMapping` type which is used in other process tracker workflows as well, I'd prefer to take this up in a separate PR.

Yes, it's possible to have something like the suggested code:

```rust
#[derive(Debug, serde::Serialize, serde::Deserialize)]
struct RetryMapping {
    pub start_after: i32,
    // Each `Vec` element is [frequency, count]
    pub frequencies: Vec<[i32; 2]>,
}


fn main() {
    let json = r#"{ "start_after": 30, "frequencies": [ [300, 2], [600, 5], [3600, 5], [21600, 3] ] }"#;
    
    let mapping: RetryMapping = serde_json::from_str(json).unwrap();
    println!("{mapping:#?}");
}
```

I've confirmed on the Rust playground that this works. But one could easily end up providing the values in the opposite order: `[count, frequency]` and nothing would prevent them from doing it.